streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -163,8 +163,13 @@ private void runSimpleCopyTest(final int numberOfRestarts,
                     Serdes.LongSerde.class.getName(),
                     new Properties() {
                         {
-                            put(StreamsConfig.consumerPrefix(ConsumerConfig.MAX_POLL_RECORDS_CONFIG), 1);
                             put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, StreamsConfig.EXACTLY_ONCE);
+                            put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0);
```

Comment:
Why do we want do disable caching for this test now? This seems to be a change in the test itself rather a set setup change?

Reply:
I'm actually not changing the behavior :) The original `getStreamsConfig()` with did lots of hard-coded customization here: https://github.com/apache/kafka/pull/5488/files/f5d72d7579b31fc6bc793767161f0312457a29db#diff-483f89729093ad48aa4b2451decc095eL48

Which I've removed, the motivation is to keep the util functions more intuitive: now we have a few overloads which all calls the underlying

```
Properties getStreamsConfig(final String applicationId,
                                               final String bootstrapServers,
                                               final String keySerdeClassName,
                                              final String valueSerdeClassName,
                                              final Properties additional)
```

which does not make any customization any more. And because of that I moved all the customizations in the caller.

Reply:
Ack. Thanks for clarification.